### ISSUE TITLE
Top 10 most watched Director / Actor / Actress

### ISSUE THREAD

**Issue by MaxHasADHD** [trakt/api-help#100]
I'd love to see an endpoint to get a top 5 or 10 director / actor / actress. I want to show this data to my users, but at the moment I can't do that unless I pull cast and crew for every movie on their account and then tally the number of characters a certain actor / director has. This could take awhile for a mobile app and sounds like it would be a nightmare for the API if an app loaded that much data for all users, so I'd love if there was an endpoint to expose cached data that is available on the stats page already perhaps

**Comment by rectifyer** [comment]
Yeah, please don't pull that much data 😄 That wouldn't be good for Trakt or your app!

Are you referring to the year in review on Trakt where we have the most watched actors for each year? We don't expose that through the API since it is limited to VIP only and we don't currently have any VIP only API methods.

**Comment by MaxHasADHD** [comment]
I was thinking of overall data for all movies, if that was known already on Trakts end.

**Comment by rectifyer** [comment]
We don't currently have that data. I'm not sure we can easily do that, it would be a ton of data to crunch to find the most watched actors globally for all movies and for all Trakt users.

**Comment by MaxHasADHD** [comment]
Sorry, I meant top 10 for the current user signed in, but for all years, or a specified one. But as you mentioned that is a VIP feature
